frontends/pytorch/test/module_import/conv2d.py

```diff
@@ -0,0 +1,41 @@
+# -*- Python -*-
+# This file is licensed under a pytorch-style license
+# See frontends/pytorch/LICENSE for license information.
+
+import typing
+
+import torch
+import torch_mlir
+
+# RUN: %PYTHON %s | npcomp-opt | FileCheck %s
+
+mb = torch_mlir.ModuleBuilder()
+
+class TestModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+        self.conv1 = torch.nn.Conv2d(in_channels=4, out_channels=4, kernel_size=(3, 3))
```

Comment:
you shouldn't need to add torch.nn.Conv2D in any of the test cases (it happens to be a good "integration test" that tickles the two separate issues; expect a model curriculum integration test store I'm still coming up with).